hs_core/models.py

```diff
@@ -240,6 +240,14 @@ def page_permissions_page_processor(request, page):
     """Return a dict describing permissions for current user."""
     from hs_access_control.models.privilege import PrivilegeCodes
 
+    def get_relation_version_res_url(resource, rel_type):
+        relation_meta_obj = resource.metadata.relations.filter(type=rel_type).first()
+        if relation_meta_obj is not None:
+            version_res_url = relation_meta_obj.value.split(',')[-1]
```

Comment:
Line 246 means the value of the metadata contains multiple items separated by ```,``` with the last item being the resource id. I don't quite understand what this function does. Since rel_type is a parameter, it seems this function could apply to multiple relation types, not specifically related to version relation, right? It'd be better to add comments to this function to elaborate a bit what it does.

Reply:
I understand what this function does after reviewing a bit more down. I still think it is better to add comments to it.

Reply:
Added comments